_posts/2022-03-10-jump-1.0.md

```diff
@@ -0,0 +1,419 @@
+[exa-scale metabolic modeling](https://github.com/LCSB-BioCore/COBREXA.jl),
+[convex programming](https://github.com/jump-dev/Convex.jl) and much more.
+
+## A brief history of JuMP
+
+* 09 Oct 2012 - First [commit](https://github.com/jump-dev/JuMP.jl/commit/688b732c13abf7182e438ca22b0b00890a08be63) to JuMP.jl repo
+* 24 Oct 2013 - [JuMP 0.1.1](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.1.1) - First public release
+* 15 Dec 2013 - [JuMP 0.1.1](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.2.0) - First support for solver callbacks
+* 02 May 2014 - [JuMP 0.1.1](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.5.0) - First support for nonlinear optimization
+* 16 Mar 2015 - First JuMP paper published: [Computing in Operations Research Using Julia](https://pubsonline.informs.org/doi/abs/10.1287/ijoc.2014.0623?journalCode=ijoc)
+* 26 Jun 2015 - First [JuMP presentation in JuliaCon 2015](https://youtu.be/7LNeR299q88)
+* 31 Aug 2015 - [JuMP 0.10.0](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.10.0) - First support for semidefinite optimization
+* 15 Dec 2016 - [INFORMS Computing Society (ICS) Prize](https://connect.informs.org/computing/awards/ics-prize) (290 stars)
+* 05 May 2017 - Second JuMP paper published: [JuMP: A Modeling Language for Mathematical Optimization](https://epubs.siam.org/doi/abs/10.1137/15M1020575?journalCode=siread)
+* 12 Jun 2017 - [First JuMP-dev workshop](/meetings/mit2017/) in Cambridge, USA
+* 27 Jun 2018 - [Second JuMP-dev workshop](/meetings/bordeaux2018/) in Bordeaux, France
+* 06 Aug 2018 - [JuMP joins NumFOCUS as a sponsored project](https://numfocus.org/blog/optimization-modeling-language-jump-joins-numfocus-sponsored-projects)
+* 15 Feb 2019 - [JuMP 0.19.0](https://github.com/jump-dev/JuMP.jl/releases/tag/v0.19.0) - Migration from [MathProgBase](https://github.com/JuliaOpt/MathProgBase.jl) to [MathOptInterface](https://github.com/jump-dev/MathOptInterface.jl)
+* 12 Mar 2019 - [Third JuMP-dev workshop](/meetings/santiago2019/) in Santiago, Chile
+* 23 Jan 2020 - First 1000 stars in the JuMP.jl repository on GitHub
+* 21 Oct 2020 - [Governance structure formalized](/pages/governance/)
+* 22 Feb 2021 - [NumFOCUS signs agreement with MIT provide ongoing maintenance and support](/announcements/2021/02/22/agreement/)
+* 23 Jul 2021 - JuMP wins the [Beale--Orchard-Hays prize](http://www.mathopt.org/?nav=boh_2021) from the Mathematical Optimization Society
+* 28 Jul 2021 - [Fourth JuMP-dev workshop](/meetings/juliacon2021/)
+* 22 Oct 2021 - Third JuMP paper published: [MathOptInterface: a data structure for mathematical optimization problems](https://pubsonline.informs.org/doi/10.1287/ijoc.2021.1067)
+* 21 Feb 2022 - [NumFOCUS signs agreement with LANL to improve nonlinear support](/announcements/2021/02/22/agreement/)
+
+## Why 1.0 and how will future versions work?
+
+For some readers, reaching "1.0.0" may signal that JuMP is only now at the point
```

Comment:
Show not tell. I'd describe some examples in the introduction part of the blog post supporting the statement, "JuMP has been mature for a long time, and is widely used in a range of production applications". Then we don't have to repeat this here. Instead say that the 1.0 release marks the completion of the 1.0 roadmap (https://jump.dev/JuMP.jl/v0.22/developers/roadmap/#JuMP-1.0) and the transition to semantic versioning.